ternaryCheckAccess.js

```diff
@@ -0,0 +1,19 @@
+// Переписать if с помощью тернарного оператора
+
+const user = {
+  age: 18,
+  paid: true,
+  blocked: false,
+  badUsername: false,
+  isAdmin: false
+};
+
+function ternaryCheckAccess(user) {
+  user.isAdmin && user.age >= 18 && user.age <= 35
+    ? console.log('Welcome admin!')
+    : user.age >= 18 && user.age <= 35 && user.paid && !user.blocked && !user.badUsername
+      ? console.log('Welcome user!')
+      : console.log('Access denied :(');
```

Comment:
Для повышения читаемости кода, при использовании тернарных операторов, можно использовать переменные*:
```suggestion
    const isAgeOk = user.age >= 18 && user.age <= 35;
    const isAdmin = user.isAdmin;
    const paid = user.paid;
    const blocked = user.blocked;
    const badUsername = user.badUsername;
    return isAgeOk && (isAdmin || paid && !blocked && !badUsername) ? true : false;
```
Вместо блока с переменными еще можно использовать деструктуризацию объекта**:
```
function ternaryCheckAccess({ age, isAdmin, paid, blocked, badUsername }) {
```

*не обязательно, но лучше делать тернарники максимально информативными
**не обязательно, но тоже повышает читабельность кода